diff --git a/node_modules/react-native-draggable/Draggable.tsx b/node_modules/react-native-draggable/Draggable.tsx
index 363ab4a..d4f8b39 100644
--- a/node_modules/react-native-draggable/Draggable.tsx
+++ b/node_modules/react-native-draggable/Draggable.tsx
@@ -125,7 +125,7 @@ export default function Draggable(props: IProps) {
     (e: GestureResponderEvent, gestureState: PanResponderGestureState) => {
       isDragging.current = false;
       if (onDragRelease) {
-        onDragRelease(e, gestureState);
+        onDragRelease(e, gestureState, getBounds());
         onRelease(e, true);
       }
       if (!shouldReverse) {
